src/synapse/azext_synapse/custom.py

```diff
@@ -60,15 +81,20 @@ def list_spark_session_jobs(cmd, client, workspace_name, spark_pool_name, from_i
     return client.list(workspace_name, spark_pool_name, from_index, size, detailed)
 
 
-def create_spark_session_job(cmd, client, workspace_name, spark_pool_name, driver_memory, driver_cores,
-                             executor_memory, executor_cores, num_executors, job_name=None, file=None, class_name=None,
-                             args=None, jars=None, files=None, archives=None, conf=None,
-                             tags=None, detailed=True):
+def create_spark_session_job(cmd, client, workspace_name, spark_pool_name, job_name, executor_size, executors,

```

Comment:
it is because  the arg_type is name_type of job_name.  It will automacically change job_name to name and add a short option "-n"

---
In reply to: [418088022](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418088022) [](ancestors = 418088022)

Reply:
And I think that --name is ok. 

---
In reply to: [418088022](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418088022) [](ancestors = 418088022)